src/main/kotlin/AppModule.kt

```diff
@@ -50,75 +42,15 @@ fun Application.module() {
         }
         route("/") {
             get {
-                call.respond(ThymeleafContent("index", mapOf("obj" to ""), locale = Locale("vi")))
+                call.respond(ThymeleafContent("index", mapOf("obj" to "")))
             }
         }
```

Comment:
I think it's possible (and maybe preferred) to change this to
get("/") {
...
}
instead of
route("/") {
    get {
        ....
    }
}